src/RecurlyUrlManager.php

```diff
@@ -25,6 +25,6 @@ public function hostedUrl($path = '', $subdomain = NULL) {
       $subdomain = \Drupal::config('recurly.settings')->get('recurly_subdomain');
     }
 
-    return \Drupal\Core\Url::fromUri('https://' . $subdomain . '.recurly.com/' . $path)->getUri();
+    return \Drupal\Core\Url::fromUri('https://' . $subdomain . '.recurly.com/' . $path);
```

Comment:
Ah so this is where you are just returning the object and this must help in other locations, but then you have to compensate by rendering this into a string in all of those other places?